packages/create-next-app/templates/index.ts

```diff
@@ -179,69 +177,70 @@ export const installTemplate = async ({
       start: 'next start',
       lint: 'next lint',
     },
+    /**
+     * Default dependencies.
+     */
+    dependencies: {
+      react: 'latest',
+      'react-dom': 'latest',
+      next: process.env.NEXT_PRIVATE_TEST_VERSION ?? 'latest',
+    },
+    devDependencies: {},
   }
-  await fs.promises.writeFile(
-    path.join(root, 'package.json'),
-    JSON.stringify(packageJson, null, 2) + os.EOL
-  )
-
-  /**
-   * These flags will be passed to `install()`, which calls the package manager
-   * install process.
-   */
-  const installFlags = { packageManager, isOnline }
-
-  /**
-   * Default dependencies.
-   */
-  const dependencies = [
-    'react',
-    'react-dom',
-    `next${
-      process.env.NEXT_PRIVATE_TEST_VERSION
-        ? `@${process.env.NEXT_PRIVATE_TEST_VERSION}`
-        : ''
-    }`,
-  ]
 
   /**
    * TypeScript projects will have type definitions and other devDependencies.
    */
   if (mode === 'ts') {
-    dependencies.push(
-      'typescript',
-      '@types/react',
-      '@types/node',
-      '@types/react-dom'
-    )
+    packageJson.devDependencies = {
+      ...packageJson.devDependencies,
+      typescript: 'latest',
+      '@types/react': 'latest',
+      '@types/node': 'latest',
+      '@types/react-dom': 'latest',
+    }
   }
 
-  /**
-   * Add Tailwind CSS dependencies.
-   */
+  /* Add Tailwind CSS dependencies. */
   if (tailwind) {
-    dependencies.push('tailwindcss', 'postcss', 'autoprefixer')
```

Comment:
Why was `autoprefixer` removed?

Reply:
See https://github.com/vercel/next.js/pull/55725

Reply:
Let's keep it as two separate PRs - one is changing to devDeps and the other is removing autoprefixer